src/Ball.js

```diff
@@ -25,16 +27,19 @@ export class Ball {
     this.hasCollision = true
     this.survivor = false
     this.hasAppInstalled = hasAppInstalled
+    this.contacts = []
```

Comment:
Potresti auitarmi a capire contacts? Da quello che visto dal codice sarebbe un peopleEnteredInContact, corretto?

Reply:
yes, `contact list` like `contact-tracing`. It's the list of other balls this ball came in touch while it's sick.